presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -381,6 +382,30 @@ public Connection getConnection(JdbcOutputTableHandle handle)
         return driver.connect(handle.getConnectionUrl(), toProperties(handle.getConnectionProperties()));
     }
 
+    @Override
```

Comment:
Put this in the client in each plugin. That's why we have this: to allow overriding in a plugin with database specific code.